content/01.main_content.md

```diff
@@ -1,4 +1,15 @@
-## Abstract {.page_break_before}
 
-
-Add text here, example citation [@https://www.theguardian.com/world/2016/may/25/enduring-whiteness-of-american-journalism; @https://medium.com/ladybits-on-medium/i-analyzed-a-year-of-my-reporting-for-gender-bias-and-this-is-what-i-found-a16c31e1cdf#.44nchdhay]
\ No newline at end of file
+Computationally-derived predictions allow for diversity audits on a scale that would not be possible otherwise.
+  Numerous tools exist to probabilistically infer gender, nationality, and ethnicity information using only the feature most likely to be non-missing in a relevant dataset: an individual's name. <decide what to cite here based on how many sources we have left>
+  Most of these models are highly scalable, allowing auditors to define the scope of their target group and background population as broadly as needed.
+  
+Prediction models are not a panacea, however; several factors limit both their accuracy at recapitulating self-reported information and their ability to address the underlying motivating questions of diversity audits.
+  For an example of the former, gender associations of a given name can vary by culture, potentially biasing gender predictions where additional information is not available [@https://aclanthology.org/U14-1021].
+  For an example of the latter, most gender prediction models are trained on binary gender labels, which occludes assessing the representation of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
+  Proxy predictions of ethnicity via name origin are more difficult still; the choice of categories to probabilistically predict on is non-trivial and difficult to define. 
+  And crucially for diversity audits, there is no one-to-one mapping between having a name from a linguistic group and belonging to a minoritized or underrepresented group.
+  Colonialism, immigration, and structural racism have affected most groups' linguistic history and inclusion or exclusion from scientific communities in complex ways that are nearly impossible to parse from names alone. 
+  For instance, these classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007]. 
+  If a target group and a background population had identical probabilistic proportions of Hispanic names, but this proportion in the target group was driven by individuals from Spain and Portugal, underrepresentation of Latin American scientists would then go unnoticed in an exclusively computational analysis.
```

Comment:
i think you can remove "probabilistic"

Reply:
Suggested edits:
If a target and background population had identical proportions of Hispanic names, but the target group primarily consisted of individuals from Spain and Portugal, underrepresentation of Latin American scientists would go unnoticed in exclusively computational analyses.

Reply:
That's much better, you're right.